pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -864,7 +880,7 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 					Secure:       true,
 				},
 				{
-					Label:        "Alert API Url",
+					Label:        "Alert API URL",
```

Comment:
A quick unobtrusive change - pay attention that only labels are changed, some properties suffer from a similar problem but those require a wider change, they remain untouched.